### ISSUE TITLE
System.ComponentModel.Tests.CategoryAttributeTests.Ctor_Category failed in CI

### ISSUE THREAD

**Issue by v-haren** [dotnet/corefx#21042]
failed test: System.ComponentModel.Tests.CategoryAttributeTests.Ctor_Category 

detail: https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netcoreapp_rhel7.2_release/77/testReport/System.ComponentModel.Tests/CategoryAttributeTests/Ctor_Category_category____Default____expectedIsDefaultAttribute__True_/

MESSAGE:
~~~
Assert.Equal() Failure\n ↓ (pos 0)\nExpected: Default\nActual: Misc\n ↑ (pos 0)
~~~
STACK TRACE:
~~~
at System.ComponentModel.Tests.CategoryAttributeTests.Ctor_Category(String category, Boolean expectedIsDefaultAttribute) in /mnt/resource/j/workspace/dotnet_corefx/master/outerloop_netcoreapp_rhel7.2_release/src/System.ComponentModel.Primitives/tests/CategoryAttributeTests.cs:line 38
~~~

**Comment by hughbe** [comment]
Fixed in #21041

**Comment by v-haren** [comment]
failed again here: https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netcoreapp_centos7.1_debug/78/testReport/System.Security.Cryptography.X509Certificates.Tests/ChainTests/VerifyWithRevocation/
